dashboard/templates/frontend/index.html

```diff
@@ -12,4 +12,5 @@
     <div id="root"></div>
     <!-- Webpack rendered JS -->
     {% render_bundle 'main' 'js' %}
+    {% render_bundle 'vendors' 'js' %}
 {% endblock %}
```

Comment:
Nit: Add newline to end of file.  I know it's not a change made for this PR, but it's an easy thing to catch and correct.